apps/blockchain/test/blockchain/block_test.exs

```diff
@@ -337,11 +337,22 @@ defmodule Blockchain.BlockTest do
 
       assert Enum.count(block.transactions) == 1
 
-      expected_receipt = %Transaction.Receipt{
-        bloom_filter: "",
+      expected_receipt = %Blockchain.Transaction.Receipt{
+        bloom_filter:
+          <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Is there an easier way to represent this? Maybe something like `<<0::2048>>`? (not sure if this is something like `256 * 8 = 2048` or something else. I'm just guessing here.)

Reply:
@germsvel I added default value to the `Receipt` struct